css/styles.css

```diff
@@ -0,0 +1,399 @@
+
+/****************************************************************/
+
+html {
+    background-color: #0D0D11;
+}
+
+
+h1,h2,h3 {    
+    font-family: Bakbak one, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #fff;
+}
+
+
+p {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    color: #FFFFFF;
+}
+
+body {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.hidden {
+    display: none;
+}
+
+/*colored text in the headers*/
+
+
+.text-label, .colored-text {
+    color: #21e786;
+}
+
+/*header*/
+
+.main-header {
+    background-color:#040b11;
+        width: 1920px;
+        height: 108px;
+    }
+
+
+.header-nav {
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+    color: #fff;
+    display: inline-block;
+
+}
+
+/*links*/
+
+.nav-link-hdr a:link {
+    font-family: Bakbak One, sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 150%;
+    letter-spacing: 0.1em;
+    text-transform: uppercase;
+    color: #fff;
+}
+
+
+.nav-link-hdr a:hover {
+    color:#21e786;
+    text-decoration: underline ;
+}
+
+
+.nav-link-hdr a:visited {
+    color:#ffffffa7;
+}
+
+/*dropdown*/
+
+.header-nav-list {
+    justify-content: flex;
+    flex-direction: row;
+    gap: 32px;
+    list-style: none;
+}
+
+.nav-link-hdr:hover > .subnav {
+    display: block;
+}
+
+.subnav {
+    display: none;
+    min-width: 100%;
+    margin: 0;
+    padding: 0;
+    list-style: none; 
+}
+
+.nav-link-hdr {
+    float: left;
+    list-style:none
+}
+
+.hdr>div { 
+    display: inline-block;
+}
+
+.header-connect-btn {
+    font-family: Bakbak One, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    text-transform: uppercase;
+    color: #FFFFFF;
+}
+
+/*********************************************************/
+
+.tagline    {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 0px;
+}
+
+.main-tagline   {
+    width: 750px;
+    height: 240px;
+    font-size: 72px;
+    line-height: 80px;
+    text-transform: capitalize;
+    text-align: left;
+    text-transform: capitalize;
+}
+
+.main-tagline>p     {
+    font-family: Poppins;
+    font-size: 20px;
+    line-height: 30px;
+}
+
+/*.numbers    {
+
+} */
+.numbers-item   { 
+    font-family: Bakbak One, sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    display: flex;
+    align-items: flex-end;
+    text-transform: capitalize;
+    color: #FFFFFF;
+    text-shadow: 0px 4px 13px #ffffff66;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
```

Comment:
Can you please tell me why you set these properties here? They don't seem to be needed.